src/controller/discoverymgr/discovery_test.go

```diff
@@ -18,7 +18,6 @@
 package discoverymgr
 
 import (
-	"log"
```

Comment:
Any reason that we need to remove `log` from `discoverymgr`, in order to fix its testing?